src/vk_mem_alloc.h

```diff
@@ -14629,7 +14628,7 @@ VkResult VmaRecorder::Init(const VmaRecordSettings& settings, bool useMutex)
     m_UseMutex = useMutex;
     m_Flags = settings.flags;
 
-#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+#if defined(_WIN32)
```

Comment:
I simplified check for windows platform here since it is used like this everywhere in the code.